system/jlib/jfile.cpp

```diff
@@ -7856,6 +7869,26 @@ static const std::array<const char*, PlaneAttributeCount> planeAttributeTypeStri
 static std::unordered_map<std::string, std::array<unsigned __int64, PlaneAttributeCount>> planeAttributesMap;
 static CriticalSection planeAttriubuteMapCrit;
 
+static IFEflags getFlagsFromStr(expertFileFlagsEnum &hasExpertFileFlags, StringBuffer &eStr)
+{
+    if (eStr.isEmpty())
+    {
+        hasExpertFileFlags = NO_MASK;
+        return IFEnone;
+    }
+
+    if (eStr.charAt(0) == '~')
```

Comment:
might you ever want to add some flags and negate others?
Might be clearer/simpler if there were 2 expert properties: expert/@enableIFileMask, expert/@disableIFileMask